Emby.Dlna/Main/DlnaEntryPoint.cs

```diff
@@ -99,6 +103,8 @@ public class DlnaEntryPoint : IServerEntryPoint, IRunBeforeStartup
             _networkManager = networkManager;
             _logger = loggerFactory.CreateLogger<DlnaEntryPoint>();
 
+            _publishedServerUrl = conf["PublishedServerUrl"];
```

Comment:
We really should be using https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.1#bind-hierarchical-configuration
```suggestion
            _publishedServerUrl = configuration["PublishedServerUrl"];
```

Reply:
agree